docs/en/observability/logs-stream.asciidoc

```diff
@@ -117,6 +99,17 @@ The following example shows the search results for logs with an `apm.error` data
 image::images/stream-logs-example.png[example search query on the logs stream page in {kib}]
 --
 
+[discrete]
+[[logs-stream-troubleshooting]]
+= Troubleshooting 
+
+If you're not seeing your log files in {kib}, check the following in the `elastic-agent.yml` file:
+
+NOTE: After installing the {agent}, you need to update the `elastic-agent.yml` in the directory described in the {fleet-guide}/installation-layout.html[installation layout].
```

Comment:
See above, the user should directly modify the elastic-agent.yml after installation.